scss/exercises.module.scss

```diff
@@ -12,7 +12,7 @@
 
 .exercise__container {
   width: 100%;
-  @media (min-width: 768px) {
+  @media (min-width: 992px) {
```

Comment:
This applies width: 75% at the lg breakpoint and above instead of the md breakpoint (see [Bootstrap breakpoints here](https://getbootstrap.com/docs/5.0/layout/breakpoints/))

Reply:
We have to do this in the SCSS file because Bootstrap doesn't have a CSS class for widths at different breakpoints.
This is equivalent to changing `md:w-3/4` to `lg:w-3/4` in Tailwind.